src/components/atoms/Badge.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+
+const getType = (type) => {
+  switch (type) {
+    case "unclicked":
+      return "flex items-center align-middle w-20 h-6 px-3.5 py-1 leading-normal text-center text-sm font-normal text-black bg-slate-200 rounded-full border border-slate-300";
+    case "onclicked":
+      return "flex items-center align-middle w-20 h-6 px-3.5 py-1 leading-normal text-center text-sm font-normal text-white bg-blue-500 rounded-full";
+    default:
+      return "";
+  }
+};
```

Comment:
```suggestion
const commonClassName = 'flex items-center align-middle w-20 h-6 px-3.5 py-1 leading-normal text-center text-sm font-normal text-black'
const getType = (type) => {
  switch (type) {
    case "unclicked":
      return `${className} bg-slate-200 rounded-full border border-slate-300`;
    case "onclicked":
      return `${className} bg-blue-500 rounded-full`;
    default:
      return "";
  }
};
```

단순한 코드이지만 정적인 값과 아닌 것(중복된 값과 아닌 값)으로 구분되는게 좋습니다.

Reply:
한 컴포넌트에 여러 스타일을 적용하는 저희 프로젝트 특성 상 유용하게 사용될 것 같습니다. 감사합니다!